app/models/service.rb

```diff
@@ -88,15 +88,15 @@ class Service < ApplicationRecord
 
   validates :name, :presence => true
 
-  default_value_for :display, false
```

Comment:
This is the line that's causing so many deprecation warnings on master.  Anything that creates a service or builds one, will get this warning.